src/main/java/org/humancellatlas/ingest/project/Project.java

```diff
@@ -59,6 +59,12 @@
     @Setter
     private String primaryWrangler;
 
+    @Setter
+    private Boolean publishedToCatalogue;
+
+    @Setter
+    private Object publicationsInfo;
```

Comment:
Should this be a `List<Object>`?

Reply:
I think this is fine, because List is also considered an object. And this makes publicationsInfo more flexible.

Reply:
Oh yes! Forgot we discussed that